### PR TITLE
Setting up Sample app with Unleash library

### PR DESCRIPTION
**Pull request by deandalm**
#### Purpose
- [X] Feature
- [ ] Bug Fix
- [ ] Other

#### Details
Added Unleash and updated the UI to show that it is fetching a feature toggle.
    
Before pulling in the source code of Unleash as a module, setting up the Sample app to use the library and confirm it is set up correctly and working.
    
Other changes:
- Updated Android Studios to 3.6.1
- Added Coroutines
- Set to compile for Java 1.8
- Updated Material colors and Launcher icon from the default ones.

#### Checklist
- [ ] Tests

### CODE REVIEW

File: sample/src/main/java/com/silvercar/sample/MainActivity.kt [hunk 1]
```diff
@@ -1,12 +1,77 @@
...
+    }
+
+    private suspend fun initUnleash(): Unleash {
+        lateinit var unleash: Unleash
+
+        return withContext(Dispatchers.IO) {
+            suspendCoroutine<Unleash> { coroutine ->
+                val config: UnleashConfig = UnleashConfig.builder()
+                    .appName(application.packageName)
+                    .unleashAPI("https://unleash.silvercar.com/api")
```
Comment (hvillanueva) [inline sample/src/main/java/com/silvercar/sample/MainActivity.kt]: should this be an ENV variable?

File: sample/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+<!--?xml version="1.0" encoding="UTF-8"?-->
```
Comment (hvillanueva) [inline sample/src/main/res/values/colors.xml]: why is this commented out?

Comment (deandalm) [reply sample/src/main/res/values/colors.xml]: That's a mistake.

Comment (deandalm) [reply sample/src/main/res/values/colors.xml]: I used this [Color Tool](https://material.io/resources/color/#!/?view.left=0&view.right=0) to generate the right colors for me and missed that the color.xml it generated made that change.

File: sample/src/main/java/com/silvercar/sample/MainActivity.kt (hunk 1 above)
Comment (deandalm) [reply sample/src/main/java/com/silvercar/sample/MainActivity.kt]: I could have put it in the `companion object` with `INTERVAL` but it's only used in this one spot right now. I'm waiting to see if I'll need it elsewhere first.

**Pull request merged**